Solutions/SlashNext/Playbooks/SlashNextWebAccessLogAssessment/azure_function_deploy.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Why do we require to create a Function App here?

Reply:
Azure function is performing following two tasks
1 Parsing URLs form raw logs which we get as input from playbook.
2 Generation of URLs and md5 mapping .